src/FakeItEasy/Core/FakeManagerProvider.cs

```diff
@@ -51,7 +51,9 @@ public IFakeCallProcessor Fetch(object proxy)
             }
         }
 
-        public void EnsureInitialized(object proxy)
+        public void EnsureInitialized(object proxy) => this.EnsureInitialized(proxy, null);
+
+        public void EnsureInitialized(object proxy, object? alias)

```

Comment:
`alias` will always be a delegate, right? So why not type it as `Delegate` and name it `delegateProxy`?